jwst/cube_build/ifu_cube.py

```diff
@@ -1351,28 +1361,21 @@ def map_fov_to_dqplane(self, this_par1, coord1, coord2, wave, roiw_ave, slice_no
                         break
                     iend = iend - 1
 
-                coord1_total = np.concatenate((coord1_start, coord1_end), axis=0)
-                coord2_total = np.concatenate((coord2_start, coord2_end), axis=0)
-
-#            coord1_start = coord1[slice == start_region]
-#            coord2_start = coord2[slice == start_region]
-
-#            coord1_end = coord1[slice == end_region]
-#            coord2_end = coord2[slice == end_region]
+                if coord1_start is not None and coord1_end is not None:
+                    coord1_total = np.concatenate((coord1_start, coord1_end), axis=0)
```

Comment:
Should there be an `else:` clause, in case `coord1_start` or `coord1_end` is still None?  A lot of variables are assigned values here.  Or you could just raise an exception if either `coord1_start` or `coord1_end` is None.

Reply:
For wavelength planes that fall in the gap between channels and there is no data in these planes the coord1_start and coord1_end will be None